ll.tm

```diff
@@ -349,6 +349,7 @@ int_type_tok : /i[0-9]+/
 'partition' : /partition/
 'personality' : /personality/
 'phi' : /phi/
+'freeze' : /freeze/
```

Comment:
The list of tokens we try to keep alphabetically sorted. I'll merge this PR, and then send a follow-up commit to sort the list and perhaps add a comment about the sorting order.